src/nodeconductor_test/helpers.py

```diff
@@ -613,6 +620,7 @@ def create_application_group(driver, project_name, category_name, resource_type_
     force_click(driver, css_selector='[visible="applications"]')
     print 'Applications tab was successfully choosen'
     time.sleep(BaseSettings.click_time_wait)
+    time.sleep(3)  #  create button isn't selected without additional time wait
```

Comment:
this random time waits looks suspicious even with comments. Please create issue to understand why site is not working without them.